### PR TITLE
rhtap: Update pipeline task name

### PR DESCRIPTION
**Pull request by Roming22**

### CODE REVIEW

**Comment by xinredhat** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-main-rhtap-installer-e2e

**Comment by xinredhat** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-main-rhtap-installer-e2e

File: ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh [hunk 1]
```diff
@@ -122,7 +122,7 @@ install_rhtap(){
     apiVersion: tekton.dev/v1
     kind: PipelineRun
     metadata:
-      generateName: redhat-trusted-application-pipeline-pe-info-
+      generateName: rhtap-pe-info-
```
Comment (flacatus) [inline ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh:125]: should  not be trusted-*

Comment (xinredhat) [reply ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh:125]: @Roming22  the value of `{{ .Chart.Name }}` is `trusted-application-pipeline` , see https://github.com/redhat-appstudio/rhtap-installer/blob/main/Chart.yaml#L13C7-L13C24. so `generateName: trusted-application-pipeline-pe-info-`

Comment (Roming22) [reply ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh:125]: @flacatus The pipeline name is changing again, cf https://github.com/redhat-appstudio/rhtap-installer/pull/103

The very long name was causing issues.

**Pull request merged**